### DISCUSSION TITLE
Update the Postgres exporter to the latest version of Postgres exporter community

### DISCUSSION THREAD

**Discussion by Jrmy2402** [grafana-cold-storage/agent#5435] [Q&A]
Hi, 
Do you forecast to update the Postgres exporter to the latest version of Postgres exporter community?
They add valuable metrics like the pg_database_size_bytes, so having this with the Grafana agent could be great.
Thanks

**Comment by lukecyca** [comment]
Also interested in this.

I've been trying to figure out why `pg_database_size_bytes` isn't being reported from `integrations/postgres` — it didn't occur to me that it simply isn't in `grafana-agent` yet. :|

**Reply by lukecyca** [reply]
https://github.com/grafana/agent/issues/5610
